docs/PROPOSAL.md

```diff
@@ -1,73 +1,12 @@
-
-There is a very clear separation between front- and back-ends, so we can delegate disjoint tasks and responsibilities. The back-end game logic can be written in C++ without any knowledge of JS (and could ideally be modular enough to couple with a command-line front-end that might be useful for development), and Greg will link the JS and C++ components together.
-
-Finally, all of these technologies are popular and industry-relevant, and the scope of the project is flexible. It should be educational!
-
-### Do you have a proof of concept?
-
-Greg built a simple proof-of-concept that illustrates the concept/workflow and relevant technologies interoperating: <https://github.com/ggorlen/wasm-click-counter>.
-
-### Division of responsibilities
-
-We can divide responsibilities up for the project roughly as follows (feel free to weigh in on your preferred role -- I'm just guessing for now):
-
-- Greg will be responsible for project management, the HTML/JS user interface, Puppeteer integration testing, and any other JS needs. He'll be responsible for keeping the project glued together. He might also be involved in the C++ engine as needed. He's happy to share any of these duties if there's interest.
-- Severin will focus on C++ game engine mechanics and some HTML/JS
-- Juan will focus on C++ game engine mechanics and assets
-- Irvin will focus on C++ game engine mechanics and assets
-
-### Further details
-
-We've decided on a one-player [puzzle game](https://en.wikipedia.org/wiki/List_of_puzzle_video_games) like Sokoban or Qualio.
-
-### Architecture diagram
-
-```
-           BACKEND (game engine)
-
-       .-------.
-       | uTest |
-       `-------`
-           |
-           v        [emscripten compilation]
-.-----------------.          |
-| C++ game engine |----------+--+
-`-----------------`             |
-                                |
-................................|.............
-                                |
-.------.   FRONTEND (browser)   |
-| User |                        |
-`------`                        |
-   |                            |
-[click events]    [API calls]   |
-   |                   |        v
-   |   .----------.~~~~+~~~>.------.
-   +-->| HTML5 UI |         | WASM |<--compiled backend engine
-       `----------`<~~~+~~~~`------`
-            ^          |
-            |     [game state]
-            |
-            |
-.----------------------.
-| Jest/Puppeteer tests |
-`----------------------`
-```
+The game engine will be written in C++ and will be compiled to [WASM](https://en.wikipedia.org/wiki/WebAssembly) with [Emscripten](https://en.wikipedia.org/wiki/Emscripten) to power a HTML5/JS/CSS user interface. Unit testing will be done with [`utest.h`](https://github.com/sheredom/utest.h) and end-to-end/integration testing will be done with [Puppeteer](https://github.com/puppeteer/puppeteer). We may use a front-end build or framework such as [React](https://reactjs.org/).
```

Comment:
Should change "utest.h" to "doctest.h" and link the new URL, here: https://github.com/doctest/doctest